_episodes/l0-course_overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Course overview"
+teaching: 5
+exercises: 0
+questions:
+- What will this course cover?
+- In what lesson-style is this course delivered?
+objectives:
+- Provide background information on course.
+keypoints:
+- Code along with the presenter.
+- Ask questions!
+---
+
+## Overview
+
+Programming as a researcher can be a very intimidating experience. It
+can feel as though your code isn't "good enough" (as judged by some
+mysterious and opaque criteria), or that you're not coding in the
+"right way". The aim of this course is to help to address some of
+these concerns through an introduction to software engineering for
+researchers. Beyond just programming, software engineering is the
+practice and principle of writing software that is correct,
+sustainable and ready to share with colleagues and the wider research
+community.
+
+Please, if you have not done so, yet, go through [the setup instructions] for the course.
+
+[the setup instructions]: {% link setup.md %}
```

Comment:
Added. I've open an issue in the intermediate course because the link was not working, indeed. 

https://github.com/ImperialCollegeLondon/intermediate_grad_school_git_course/issues/97